set.mm

```diff
@@ -5795,6 +5804,15 @@ use disjunction (although this is not required since definitions are
       ( wa wb adantr adantl bibi12d ) ADIBCEFABCJDGKDEFJAHLM $.
   $}
 
+  ${
```

Comment:
A shorter proof uses adantr adantl impbid

Reply:
I will shorten the proof accordingly (as done with ~19.28)